src/main/java/org/apache/maven/shared/incremental/IncrementalBuildHelper.java

```diff
@@ -48,6 +52,13 @@
     public static final String CREATED_FILES_LST_FILENAME = "createdFiles.lst";
     private static final String INPUT_FILES_LST_FILENAME = "inputFiles.lst";
 
+    private static final String INFO_FILENAME = "info.properties";
+
+    private static final String LAST_COMPILE_SUCCESS_TIMESTAMP = "lastCompile.success.timestamp";
+
+    private static final String LAST_TEST_COMPILE_SUCCESS_TIMESTAMP = "lastTestCompile.success.timestamp";
```

Comment:
Unless you are going to have other keys under `success` one should rather use underscores or camelCase since dots denote a hierarchy. Similar: https://issues.apache.org/jira/browse/VELOCITY-909

Reply:
why not... I don't really have strong opinion here. conventions are done by people who wrote it. so there can be so many different conventions :P. sounds really details as normally nobody will read directly those files...

Reply:
Sure, I neither have one. Just wanted to point out what alternatives are out there and how it could be misunderstood.